tests/test_validation.py

```diff
@@ -87,3 +88,63 @@ def test_invalid_kms_key(self, mock_kms):
             from cis.validation import validate
             is_valid_payload = validate(**payload)
         self.assertFalse(is_valid_payload)
+
+
+class DatabaseTest(unittest.TestCase):
+    def setUp(self):
+        # Load json with test data
+        fixtures = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'data/fixtures.json')
+        with open(fixtures) as artifacts:
+            self.test_artifacts = json.load(artifacts)
+
+        # Setup env vars with dummy AWS credentials
+        os.environ['CIS_ARN_MASTER_KEY'] = self.test_artifacts['dummy_kms_arn']
+        os.environ['AWS_DEFAULT_REGION'] = self.test_artifacts['dummy_aws_region']
+        os.environ['CIS_DYNAMODB_TABLE'] = self.test_artifacts['dummy_dynamodb_table']
+
+    @patch('cis.validation.boto3')
+    def test_store_success(self, mock_boto):
+        dynamodb_mock = Mock()
+        dynamodb_table = Mock()
+        dynamodb_table.put_item.return_value = {
+            'ResponseMetadata': {
+                'HTTPStatusCode': 200,
+            }
+        }
+
+        dynamodb_mock.Table.return_value = dynamodb_table
+        mock_boto.resource.return_value = dynamodb_mock
+
+        data = {
+            'foo': 'bar',
+            'foobar': 42
+        }
+
+        from cis.validation import store_to_vault
```

Comment:
We need this here for setUp to work correctly (we are patching env vars) in all our testcases